socorro/scripts/pubsub.py

```diff
@@ -142,7 +142,10 @@ def publish_crashid(config, args):
     topic_name = config['resource.pubsub.%s_topic_name' % queue]
     topic_path = publisher.topic_path(project_id, topic_name)
 
-    for crash_id in args.crashid:
+    if not args.crashid:
+        print('Reading crash IDs from stdin')
+    crashids_iterable = args.crashid or sys.stdin
```

Comment:
I think you got this from `fetch_crash_data.py`. There's a better way to do this that better handles the case where they didn't pass anything in--then it won't hang indefinitely. We do that in `reprocess.py`:

https://github.com/mozilla-services/socorro/blob/master/socorro/scripts/reprocess.py#L63-L71

I think `fetch_crash_data.py` should use that code. Also, maybe it makes sense to lift that to `socorro/scripts/__init__.py` so any of the scripts can use it.

We don't have to to that in this PR. I'm just thinking out loud.

Reply:
Thanks, I was looking around for a way to do that - it was weird to have ``socorro_cmd pubsub publish`` just hang waiting for input, so I added a ``print`` message. I think this is a better solution, and I'll switch.